core/src/main/kotlin/org/evomaster/core/output/OutputFormat.kt

```diff
@@ -17,7 +17,8 @@ enum class OutputFormat {
     JAVA_JUNIT_4,
     KOTLIN_JUNIT_4,
     KOTLIN_JUNIT_5,
-    JS_JEST
+    JS_JEST,
+    DOTNET_CORE_3
```

Comment:
hmmm, in .Net there can be different languages, such as F\# for example.
So, maybe we should specify `CSHARP_?`, where ? is the name of the library used for writing the test cases (eg like JUnit for JVM and Jest for JS)

Reply:
Thanks for pointing out. I changed it to CSHARP_XUNIT